package-lock.json

```diff
@@ -1229,7 +1229,7 @@
     },
     "node_modules/@clack/prompts/node_modules/is-unicode-supported": {
       "version": "1.3.0",
-      "extraneous": true,
+      "dev": true,
```

Comment:
Why is this being reverted?

Reply:
Hi, I thought I had uploaded this file too, sorry